src/users/tests.py

```diff
@@ -54,6 +47,44 @@ def test_user_username_and_email_equal(self):
         user = User.objects.get(email=user_params['email'])
         self.assertEqual(user.email, user.username)
 
+class VerifyUserTest(APITestCase):
+    verify_url = reverse('users:verify')
+
+    def setUp(self):
+        self.profile = factories.ProfileFactory()
+        self.client.force_authenticate(user=self.profile.user)
+
+    def tearDown(self):
+        User.objects.all().delete()
+        Profile.objects.all().delete()
+
+    def test_verify_only_accepts_correct_code(self):
+        user_params = {
+            'verification_code' : '0',
+        }
+
+        resp = self.client.post(
+            self.verify_url,
+            data=user_params,
+        )
+
+        self.assertEqual(self.profile.verified, False)
+
+
+    #TODO: Tests (Need to have Factories generate a Verification Code)
```

Comment:
you can pass it in as 
`self.profile = factories.ProfileFactory(verification_code='this_is_a_test_code')`